GeneratorInterface/GenFilters/plugins/MCLongLivedParticles.h

```diff
@@ -0,0 +1,55 @@
+// -*- C++ -*-
+//
+// Package:    MCLongLivedParticles
+// Class:      MCLongLivedParticles
+//
+/* 
+
+ Description: 
+Filter particles based on their minimum and/or maximum displacement on the transverse plane and optionally on their pdgIds
+To run independently of pdgIds, do not insert the particleIDs entry in filter declaration
+
+*/
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDFilter.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+//
+// class decleration
+//
+
+namespace edm {
+  class HepMCProduct;
+  class ConfigurationDescriptions;
+}  // namespace edm
+
+class MCLongLivedParticles : public edm::global::EDFilter<> {
+public:
+  explicit MCLongLivedParticles(const edm::ParameterSet&);
+  // ~MCLongLivedParticles() override;
```

Comment:
```suggestion
  ~MCLongLivedParticles() override = default;
```
(why did you remove it)